### PR TITLE
Change Partner token to OTC [#168661200]

### PR DESCRIPTION
**Pull request by vitaliypodlypnyi**

### CODE REVIEW

File: CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs [hunk 1]
```diff
@@ -176,6 +183,33 @@ private async Task<RSAParameters> GetRsaParameters()
...
+        private async Task<string> GetOneTimeCodeAsync()
+        {
+            var requestParams = new List<KeyValuePair<string, string>>
+            {
+                new KeyValuePair<string, string>("client_id", _clientId),
+                new KeyValuePair<string, string>("client_secret", _clientSecret),
+                new KeyValuePair<string, string>("response_type", "one_time_code")
+            };
+
+            var restClient = new HttpClient();
```
Comment (KaterynaZorina) [inline CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs:196]: `using` is missing.

File: CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs [hunk 2]
```diff
@@ -176,6 +183,33 @@ private async Task<RSAParameters> GetRsaParameters()
...
+
+            var request = new HttpRequestMessage(
```
Comment (KaterynaZorina) [inline CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs:198]: `using` is missing.

File: CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs [hunk 3]
```diff
@@ -176,6 +180,33 @@ private async Task<RSAParameters> GetRsaParameters()
...
+            {
+                new KeyValuePair<string, string>("client_id", _clientId),
+                new KeyValuePair<string, string>("client_secret", _clientSecret),
+                new KeyValuePair<string, string>("response_type", "one_time_code")
+            };
+
+            string responseContent;
+            using (var restClient = new HttpClient())
+            {
+                var request = new HttpRequestMessage(HttpMethod.Post, $"{_zgApiUrl}/connect/clientauthentication")
```
Comment (KaterynaZorina) [inline CredentialPostTest/CredentialPostTest/Pages/Index.cshtml.cs:196]: `HttpRequestMessage` also implements `IDisposable`.

**Comment by dennisahlin** [comment]
Why is still PartnerToken present? Shouldn't clientsecret/clientid replace it?

**Pull request merged**